
Cleanup pkg-plist a bit to get correct package w/ different options
except for FREESTYLE to reduce hunk count

--- pkg-plist.orig	2016-07-29 16:42:07.000000000 +0300
+++ pkg-plist
@@ -88,7 +88,7 @@ share/applications/blender.desktop
 %%OPENCOLORIO%%%%DATADIR%%/%%VER%%/datafiles/colormanagement/luts/spi_ocio_srgb_test.spi3d
 %%OPENCOLORIO%%%%DATADIR%%/%%VER%%/datafiles/colormanagement/luts/srgb.spi1d
 %%OPENCOLORIO%%%%DATADIR%%/%%VER%%/datafiles/colormanagement/luts/srgb_to_xyz.spimtx
-%%NLS%%%%DATADIR%%/%%VER%%/datafiles/colormanagement/luts/vd16.spi1d
+%%OPENCOLORIO%%%%DATADIR%%/%%VER%%/datafiles/colormanagement/luts/vd16.spi1d
 %%NLS%%%%DATADIR%%/%%VER%%/datafiles/fonts/bmonofont-i18n.ttf.gz
 %%NLS%%%%DATADIR%%/%%VER%%/datafiles/fonts/droidsans.ttf.gz
 %%NLS%%%%DATADIR%%/%%VER%%/datafiles/locale/ar/LC_MESSAGES/blender.mo
@@ -161,160 +161,160 @@ share/applications/blender.desktop
 %%DATADIR%%/%%VER%%/scripts/addons/animation_animall.py
 %%DATADIR%%/%%VER%%/scripts/addons/bone_selection_groups.py
 %%DATADIR%%/%%VER%%/scripts/addons/curve_simplify.py
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/__init__.py
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/engine.py
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_ashikhmin_shirley.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_ashikhmin_velvet.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_diffuse.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_diffuse_ramp.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_hair.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_microfacet.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_oren_nayar.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_phong_ramp.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_reflection.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_refraction.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_toon.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_transparent.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_util.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bssrdf.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/emissive.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/volume.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_attribute.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh_shadow.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh_subsurface.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh_traversal.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh_volume.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh_volume_all.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_curve.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_motion_curve.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_motion_triangle.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_object.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_primitive.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh_shadow.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh_subsurface.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh_traversal.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh_volume.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh_volume_all.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_triangle.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_triangle_intersect.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_volume.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_accumulate.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_bake.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_camera.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_compat_cpu.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_compat_cuda.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_compat_opencl.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_debug.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_differential.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_emission.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_film.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_globals.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_jitter.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_light.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_math.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_montecarlo.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_passes.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_branched.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_common.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_state.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_surface.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_volume.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_projection.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_queues.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_random.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_shader.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_shadow.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_subsurface.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_textures.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_types.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_volume.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_work_stealing.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/cuda/kernel.cu
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_background_buffer_update.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_data_init.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_direct_lighting.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_holdout_emission_blurring_pathtermination_ao.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_lamp_emission.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_next_iteration_setup.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_queue_enqueue.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_scene_intersect.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_shader_eval.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_shadow_blocked.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_sum_all_radiance.cl
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_background_buffer_update.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_data_init.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_direct_lighting.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_holdout_emission_blurring_pathtermination_ao.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_lamp_emission.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_next_iteration_setup.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_scene_intersect.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_shader_eval.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_shadow_blocked.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_split_common.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_sum_all_radiance.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_attribute.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_blackbody.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_brick.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_brightness.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_camera.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_checker.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_closure.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_convert.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_displace.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_fresnel.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_gamma.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_geometry.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_gradient.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_hsv.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_image.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_invert.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_light_path.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_magic.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_mapping.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_math.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_math_util.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_mix.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_musgrave.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_noise.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_noisetex.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_normal.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_ramp.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_sepcomb_hsv.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_sepcomb_vector.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_sky.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_tex_coord.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_texture.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_types.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_value.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_vector_transform.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_voronoi.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_voxel.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_wave.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_wavelength.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_wireframe.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_atomic.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_color.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_half.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_math.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_math_fast.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_transform.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_types.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/Apache_2.0.txt
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/ILM.txt
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/NVidia.txt
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/OSL.txt
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/Sobol.txt
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/readme.txt
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/osl.py
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/presets.py
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/properties.py
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/__init__.py
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/engine.py
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_ashikhmin_shirley.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_ashikhmin_velvet.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_diffuse.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_diffuse_ramp.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_hair.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_microfacet.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_oren_nayar.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_phong_ramp.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_reflection.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_refraction.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_toon.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_transparent.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bsdf_util.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/bssrdf.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/emissive.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/closure/volume.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_attribute.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh_shadow.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh_subsurface.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh_traversal.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh_volume.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_bvh_volume_all.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_curve.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_motion_curve.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_motion_triangle.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_object.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_primitive.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh_shadow.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh_subsurface.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh_traversal.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh_volume.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_qbvh_volume_all.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_triangle.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_triangle_intersect.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/geom/geom_volume.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_accumulate.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_bake.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_camera.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_compat_cpu.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_compat_cuda.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_compat_opencl.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_debug.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_differential.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_emission.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_film.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_globals.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_jitter.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_light.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_math.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_montecarlo.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_passes.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_branched.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_common.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_state.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_surface.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_volume.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_projection.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_queues.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_random.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_shader.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_shadow.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_subsurface.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_textures.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_types.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_volume.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_work_stealing.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/cuda/kernel.cu
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_background_buffer_update.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_data_init.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_direct_lighting.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_holdout_emission_blurring_pathtermination_ao.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_lamp_emission.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_next_iteration_setup.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_queue_enqueue.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_scene_intersect.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_shader_eval.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_shadow_blocked.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernels/opencl/kernel_sum_all_radiance.cl
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_background_buffer_update.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_data_init.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_direct_lighting.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_holdout_emission_blurring_pathtermination_ao.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_lamp_emission.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_next_iteration_setup.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_scene_intersect.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_shader_eval.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_shadow_blocked.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_split_common.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/split/kernel_sum_all_radiance.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_attribute.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_blackbody.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_brick.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_brightness.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_camera.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_checker.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_closure.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_convert.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_displace.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_fresnel.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_gamma.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_geometry.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_gradient.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_hsv.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_image.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_invert.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_light_path.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_magic.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_mapping.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_math.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_math_util.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_mix.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_musgrave.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_noise.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_noisetex.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_normal.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_ramp.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_sepcomb_hsv.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_sepcomb_vector.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_sky.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_tex_coord.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_texture.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_types.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_value.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_vector_transform.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_voronoi.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_voxel.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_wave.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_wavelength.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_wireframe.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_atomic.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_color.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_half.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_math.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_math_fast.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_transform.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/util_types.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/Apache_2.0.txt
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/ILM.txt
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/NVidia.txt
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/OSL.txt
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/Sobol.txt
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/license/readme.txt
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/osl.py
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/presets.py
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/properties.py
 %%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_absorption_volume.oso
 %%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_add_closure.oso
 %%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_ambient_occlusion.oso
@@ -399,8 +399,8 @@ share/applications/blender.desktop
 %%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_wireframe.oso
 %%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/oslutil.h
 %%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/stdosl.h
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/ui.py
-%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/version_update.py
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/ui.py
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/version_update.py
 %%DATADIR%%/%%VER%%/scripts/addons/development_api_navigator.py
 %%DATADIR%%/%%VER%%/scripts/addons/development_icon_get.py
 %%DATADIR%%/%%VER%%/scripts/addons/game_engine_publishing.py
@@ -1133,7 +1133,7 @@ share/applications/blender.desktop
 %%DATADIR%%/%%VER%%/scripts/templates_py/ui_previews_dynamic_enum.py
 %%PORTDOCS%%%%DOCSDIR%%/GPL-license.txt
 %%PORTDOCS%%%%DOCSDIR%%/GPL3-license.txt
-%%NLS%%%%PORTDOCS%%%%DOCSDIR%%/LICENSE-bfont.ttf.txt
+%%PORTDOCS%%%%DOCSDIR%%/LICENSE-bfont.ttf.txt
 %%NLS%%%%PORTDOCS%%%%DOCSDIR%%/LICENSE-bmonofont-i18n.ttf.txt
 %%NLS%%%%PORTDOCS%%%%DOCSDIR%%/LICENSE-droidsans.ttf.txt
 %%PORTDOCS%%%%DOCSDIR%%/jemalloc-license.txt
